### PR TITLE
Release 4.1.0

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.1.0 (2020-08-10)
+
+- Added the ability for the phone field component to have its country value set programatically. [#51](https://github.com/blackbaud/skyux-phone-field/pull/51)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: I'd tweak this to: 
```Added the ability to programatically set the phone field component's country value.```

No worries if you don't think it's worth the effort though. I'll go ahead and approve the PR and leave it up to you.

**Review by blackbaud-johnly — DISMISSED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-release-4.1.0](https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-release-4.1.0)

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Done

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-release-4.1.0](https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-release-4.1.0)

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**